sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -208,27 +208,27 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-messaging-eventgrid</artifactId>
-        <version>4.20.1</version>
+        <version>4.21.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-messaging-eventhubs</artifactId>
-        <version>5.18.0</version>
+        <version>5.18.1</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
-        <version>1.19.0</version>
+        <version>1.19.1</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-messaging-servicebus</artifactId>
-        <version>7.15.0</version>
+        <version>7.15.1</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-messaging-webpubsub</artifactId>
-        <version>1.2.11</version>
+        <version>1.2.12</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
```

Comment:
Has there been a patch release for this one as well?

Reply:
Yes, https://mvnrepository.com/artifact/com.azure/azure-messaging-webpubsub/1.2.12